app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -221,6 +222,7 @@ public void onDetachedFromRecyclerView(RecyclerView recyclerView) {
     public int getItemViewType(int position) {
         ChatMessage item = getItem(position);
 
+        assert item != null;
```

Comment:
Never use assert in production code

Reply:
We don't need to throw exceptions. We need to check if they are null and handle it gracefully

If they should not be null and they are, then well any other exception is not much better than NullPointerException

Mostly, exceptions are good in service layer, never in view layer